api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/repositories/final/blm-final-results.repository.ts

```diff
@@ -0,0 +1,10 @@
+import { ExecutionResult } from '@marxan/marxan-output';
+
+export abstract class BlmFinalResultsRepository {
```

Comment:
[blocker] seems to be a port but is within adapters? What happened to original ports?